.github/workflows/pr_pre-commit.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.11", "3.12"]
+        python-version: ["3.11", "3.12", "3.13"]
```

Comment:
suggestion: We could use the [`stackabletech/actions/run-pre-commit`](https://github.com/stackabletech/actions/tree/main/run-pre-commit) action here. But it doesn't offer a huge benefit compared to the current solution.

Reply:
[done](https://github.com/stackabletech/beku.py/pull/33/commits/c3650ef58c531582b6018821c0e55e56daa0d6e8)